src/hooks/useQuiz.ts

```diff
@@ -52,6 +52,7 @@ const useQuiz: UseQuiz = (quizOptions) => {
         currentQuestion: quizApiState.quizCurrentQuestion,
```

Comment:
This is not related to this line but can you also update `src/stories/Quiz/Hooks/index.tsx` and `src/stories/Quiz/Hooks/Docs/markdown/CodeExample.md` to match the new format?

Reply:
[9e749d6](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/9e749d6bba3f4e510b6fd98bbe0e260357f63cf4)